lib/moneta/proxy.rb

```diff
@@ -17,6 +17,11 @@ def key?(key, options = {})
       adapter.key?(key, options)
     end
 
+    # (see Defaults#each_key)
+    def each_key
+      adapter.each_key { yield }
```

Comment:
This method doesn't handle the case where no block is given, and it should also return `self` when a block is given.